pkg/apis/core/v1alpha1/helper/shootstate_lister.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Wouldn't be better to name `shootstate_list.go` instead of `shootstate_lister.go`. I would rather consider it as client-go lister when I see lister suffix of the file. WDYT?

Reply:
makes sense